### ISSUE TITLE
All windows not present after reopening WKit, despite checkmark in "View" tab showing they are

### ISSUE THREAD

**Issue by 86maylin** [WolvenKit/WolvenKit#1047]
**Describe the bug**
All windows disappeared after reopening WKit, despite checkmark in "View" tab showing they should be there. To call them up again you'll have to uncheck them and recheck them to make them appear. 

**To Reproduce**
1. Open an old project
2. Open some windows in the "View" tab
3. Close WKit
4. Reopen WKit and reopen project
5. It's emptyyyyyy

**Expected behavior**
It should save the project windows, or at least make it so the indicator in "View" tab is really what's shown. 

**Screenshots**
![image](https://user-images.githubusercontent.com/20621809/210163275-4c6020bc-5979-4caa-8c2a-c7c7c673595b.png)

Tested on nightly 12-31

**Comment by rfuzzo** [comment]
Yeah thank you, can you try clicking "reset layout"? Either it will just work, or you need to enable the individual tool windows again in the View > Menu.

I'll try to fix the bug though regardless.

**Comment by 86maylin** [comment]
> can you try clicking "reset layout"? Either it will just work, or you need to enable the individual tool windows again in the View > Menu.

Reset layout works in bringing the windows back, tho as it says, it resets my layout so everything's back to its default position, not ideal. :P

**Comment by rfuzzo** [comment]
Well you only have to do it once…

**Comment by 86maylin** [comment]
> Well you only have to do it once…

I'll have to reset layout and reorder all my windows after each WKit restart no? :fuzzo_fix: pls

**Comment by rfuzzo** [comment]
What? No. Don’t be silly. You need only to do it once. The last nightly introduced a breaking change where you need to reset the layout *once* - and after that it just works.

I’ll add a bugfix at some point where the layout gets reset automatically, and it doesn’t show an empty screen. You will still have to rearrange your windows *once*.

**Comment by 86maylin** [comment]
Hmm.....when I reset my layout and close and reopen WKit it's empty again...

**Comment by rfuzzo** [comment]
T_T

**Comment by 86maylin** [comment]
Ok so I vaguely remembered about the save layout button so I pressed saved layout to project after resetting and now it works after restart. xD

**Comment by rfuzzo** [comment]
Hmm, it should also work without that… but I’ll take a look. Thanks!

**Comment by rfuzzo** [comment]
this should be fixed now,
